neue-juristische-wochenschrift.csl

```diff
@@ -109,7 +109,7 @@
             <text macro="firstpage-locator"/>
           </group>
         </if>
-        <else-if type="article-newspaper">
+        <else-if type="article-newspaper post-weblog webpage song">
```

Comment:
this needs match="any" (if/else-if default to match="all" which would never evaluate to true in this case.)